src/ExamsPage/data/reducer.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  currentExam: null,
+  examsList: [],
+};
+
+export const examId = (exam) => `exam-${exam.id}`;
+
+const slice = createSlice({
```

Comment:
client state for exams. This keeps the state the UI reads from limited to only objects/attributes we care about vs. storing the entire network response in state.